subprojects/docs/src/samples/java/jvm-multi-project-with-additional-test-types/groovy/buildSrc/src/main/groovy/myproject.java-conventions.gradle

```diff
@@ -1,6 +1,9 @@
+// tag::apply-jvm-test-suite[]
 plugins {
     id 'java'
+    id 'jvm-test-suite'
```

Comment:
I think it's OK to show how to apply this plugin, but our samples/snippets don't need to apply this plugin since it's a part of the java plugin already.